### PR TITLE
chapter 4

### PR DESCRIPTION
**Pull request by Dmitriy-Romanchuk**

### CODE REVIEW

File: chapter_4_practic/task1.cpp [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+{
+	struct Phone
+	{
+		int code;
+		int exchange;
+		int number;
+	};
+
+	Phone my_phone {212, 870, 8491};
+	Phone your_phone;
```
Comment (zefir-o) [inline chapter_4_practic/task1.cpp:24]: initialize variables always

File: chapter_4_practic/task1.cpp [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+	std::cout << "Enter your area code, exchange, and number: XXX XXX XXXX: ";
+	std::cin >> your_phone.code >> your_phone.exchange >> your_phone.number;
+
+	std::cout << "My number is: (" << my_phone.code << ") " << my_phone.exchange << "-" << my_phone.number << std::endl;
```
Comment (zefir-o) [inline chapter_4_practic/task1.cpp:29]: what if the typed number contains 111 000 0001
the output will be incorrect

File: chapter_4_practic/task1.cpp [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+// My number is(212) 767 - 8900
+// Your number is(415) 555 - 1212
+
+#include <iostream>
+
+int main()
+{
+	struct Phone
+	{
+		int code;
```
Comment (zefir-o) [inline chapter_4_practic/task1.cpp]: all structure members should have default initializers

File: chapter_4_practic/task6.cpp [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+// accountant, executive, researcher): a
+// Employee type is accountant.
+// You’ll probably need two switch statements : one for inputand one for output.
+
+#include <iostream>
+
+int main()
+{
+	char ch = ' ';
+	enum etype { laborer, secretary, manager, accountant, executive, researcher };
```
Comment (zefir-o) [inline chapter_4_practic/task6.cpp]: prefer the usage of 
enum class
https://en.cppreference.com/w/cpp/language/enum

**Review by zefir-o — APPROVED** [review]

File: chapter_4_practic/task1.cpp (hunk 1 above)
Comment (Dmitriy-Romanchuk) [reply chapter_4_practic/task1.cpp:24]: fixed in https://github.com/Dmitriy-Romanchuk/master/pull/4/commits/9d9d2ec85e205856ca38d8fcb47a519931b08150

File: chapter_4_practic/task1.cpp (hunk 2 above)
Comment (Dmitriy-Romanchuk) [reply chapter_4_practic/task1.cpp:29]: fixed in https://github.com/Dmitriy-Romanchuk/master/pull/4/commits/9d9d2ec85e205856ca38d8fcb47a519931b08150#diff-8d44ec055bab280af13c782626e4996fefc1313e0525534b550c72695088963b

File: chapter_4_practic/task1.cpp (hunk 3 above)
Comment (Dmitriy-Romanchuk) [reply chapter_4_practic/task1.cpp]: fixed in https://github.com/Dmitriy-Romanchuk/master/pull/4/commits/9d9d2ec85e205856ca38d8fcb47a519931b08150

File: chapter_4_practic/task6.cpp (hunk 4 above)
Comment (Dmitriy-Romanchuk) [reply chapter_4_practic/task6.cpp]: fixed in https://github.com/Dmitriy-Romanchuk/master/pull/4/commits/2048c13f717dd24c8d8d80020596b020faed2a90

**Comment by zefir-o** [comment]
Looks good, you could submit

**Pull request merged**